src/nwm_routing/src/nwm_routing/output.py

```diff
@@ -213,6 +214,24 @@ def nwm_output_generator(
         LOG.debug("writing CSV file took %s seconds." % (time.time() - start))
         # usgs_df_filtered = usgs_df[usgs_df.index.isin(csv_output_segments)]
         # usgs_df_filtered.to_csv(output_path.joinpath("usgs_df.csv"))
+        
+    if chano:
+        
+        LOG.info("- writing t-route flow results at gage locations to CHANOBS file")
+        start = time.time()
+                
+        nhd_io.write_chanobs(
+            Path(chano['chanobs_output_directory'] + chano['chanobs_filepath']), 
+            flowveldepth, 
+            link_gage_df, 
+            t0, 
```

Comment:
The model output looks like this on my end just for reference. 

Dimensions:         (feature_id: 145, reference_time: 1, time: 936)
Coordinates:
  * time            (time) datetime64[ns] 2018-09-01T00:05:00 ... 2018-09-02
  * reference_time  (reference_time) datetime64[ns] 2018-09-01
  * feature_id      (feature_id) int32 3350287 3350831 ... 166737579 933020162
Data variables:
    streamflow      (time, feature_id) float32 ...
Attributes:
    model_initialization_time:  2018-09-01_00:00:00
    model_output_valid_time:    2018-09-01_00:05:00